recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ source:
   sha256: {{ sha256 }}
   
 build:
-  number: 0
+  number: 1
```

Comment:
This can stay `0` as we are doing a new release and there are no broken builds that we are fixing yet.